python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -826,6 +826,28 @@ def ElementwiseMulTensorIntModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwiseMishModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.mish(x)
+
+
+@register_test_case(module_factory=lambda: ElementwiseMishModule())
+def ElementwiseMishModule_basic(module, tu: TestUtils):
+    module.forward(2 * tu.rand(5, 3, low=-3.0, high=3.0) - 0.5)
```

Comment:
I think my comment might not have been clear. Sorry about that. What I meant was that we should remove the `2 *` and the `- 0.5` and simply use the `low` and `high` to specify the range. (the `2 *` and `- 0.5`) are simply modifying the range to be twice as big and shifted 0.5 to the left, which are things that can be achieved with with `low` and `high`.

In short, this line should be

```python
module.forward(tu.rand(5, 3, low=-3.0, high=3.0))
```

Reply:
Oh all right, I understand what you meant to say now and have updated the test case - you may re-review. :) Thanks.